docs/src/conf.py

```diff
@@ -0,0 +1,103 @@
+# Configuration file for the Sphinx documentation builder.
```

Comment:
It would be nice if the sidebar contained "Stuck? Ask us on the discussion board. We're happy to help!", as the GitHub pages version does, but I don't think this is possible without modifying the sphinx theme itself?